src/ast/ast.h

```diff
@@ -214,7 +214,7 @@ class Offsetof : public Expression {
   std::vector<std::string> field;
 };
 
-class MapDeclStatement : public Expression {
+class MapDeclStatement : public Node {
```

Comment:
@amscanne Shouldn't this be a `Statement` instead of a `Node`?  Second question: should we just remove `Statement` as it seems to be the same as `Node`.

Reply:
No, statements appear in blocks. This isn't something that can be parsed as part of a block. Perhaps it should be named `MapDecl`, but alas. This stuff will probably keep changing quite a lot, no need to fuss, I just noticed that this wasn't quite right and fired off a PR.